great_tables/_gt_data.py

```diff
@@ -1136,6 +1144,9 @@ class Options:
     quarto_disable_processing: OptionsInfo = OptionsInfo(False, "quarto", "logical", False)
     quarto_use_bootstrap: OptionsInfo = OptionsInfo(False, "quarto", "logical", False)
 
+    def _replace(self, **kwargs: OptionsInfo) -> Self:
```

Comment:
(See comment above.) I think we can remove this, since the dataclasses.replace function works okay here.